DotNetHowToMultipleDataSources/Program.cs

```diff
@@ -0,0 +1,153 @@
+            // https://docs.microsoft.com/azure/search/search-howto-index-cosmosdb#indexing-changed-documents
+            DataSource cosmosDbDataSource = DataSource.DocumentDb(
+                name: "cosmos-db",
+                documentDbConnectionString: configuration["CosmosDBConnectionString"],
+                collectionName: "hotels",
+                useChangeDetection: true);
+            // The Cosmos DB data source does not need to be deleted if it was already created,
+            // but the connection string may need to be updated if it was changed
+            await searchService.DataSources.CreateOrUpdateAsync(cosmosDbDataSource);
+
+            Console.WriteLine("Creating Azure SQL indexer...");
+            Indexer sqlIndexer = new Indexer(
+                name: "azure-sql-indexer",
+                dataSourceName: sqlDataSource.Name,
+                targetIndexName: index.Name,
+                schedule: new IndexingSchedule(TimeSpan.FromDays(1)));
+            // Indexers contain metadata about how much they have already indexed
+            // If we already ran the sample, the indexer will remember that it already
+            // indexed the sample data and not run again
+            // To avoid this, reset the indexer if it exists
+            exists = await searchService.Indexers.ExistsAsync(sqlIndexer.Name);
+            if (exists)
+            {
+                await searchService.Indexers.ResetAsync(sqlIndexer.Name);
+            }
+
+            await searchService.Indexers.CreateOrUpdateAsync(sqlIndexer);
+
+            Console.WriteLine("Creating Cosmos DB indexer...");
+            Indexer cosmosDbIndexer = new Indexer(
+                name: "cosmos-db-indexer",
+                dataSourceName: cosmosDbDataSource.Name,
+                targetIndexName: index.Name,
+                schedule: new IndexingSchedule(TimeSpan.FromDays(1)));
+
+            // Indexers contain metadata about how much they have already indexed
+            // If we already ran the sample, the indexer will remember that it already
+            // indexed the sample data and not run again
+            // To avoid this, reset the indexer if it exists
+            exists = await searchService.Indexers.ExistsAsync(cosmosDbIndexer.Name);
+            if (exists)
+            {
+                await searchService.Indexers.ResetAsync(cosmosDbIndexer.Name);
+            }
+
+            await searchService.Indexers.CreateOrUpdateAsync(cosmosDbIndexer);
+
+            // We created two indexer with schedules, but we also
+            // want to run them immediately
+            Console.WriteLine("Running Azure SQL and Cosmos DB indexers...");
+
+            try
+            {
+                await searchService.Indexers.RunAsync(sqlIndexer.Name);
+                await searchService.Indexers.RunAsync(cosmosDbIndexer.Name);
+
+            }
+            catch (CloudException e) when (e.Response.StatusCode == (HttpStatusCode)429)
```

Comment:
@mgottein When does `RunAsync` return 429? Is that a common case?

Reply:
@HeidiSteen @brjohnstmsft  I copied this from the other indexer sample. I agree, we should save the 429 check for the common "run all the samples on free tier, why doesn't it work?" case

Reply:
@HeidiSteen This code is running an indexer, not creating an index, so the case you mentioned wouldn't apply.

@mgottein Many customers do run samples on the free tier. Are you saying we should leave the check as-is? I'm ok with that, but maybe we should add some syntactic sugar for that case to the SDK, since it is apparently common(?).

Reply:
@mgottein It would definitely have to be a separate PR, because I'm talking about changing the SDK, not the samples.